docs/config.md

```diff
@@ -0,0 +1,147 @@
+This document describes the configuration files of the different packages.
+
+## AEA config yaml
```

Comment:
@marcofavorito added full list of available configs..can still be improved but this is a start

Reply:
Excellent!